libdnf.spec

```diff
@@ -51,6 +51,7 @@ BuildRequires:  pkgconfig(cppunit)
 BuildRequires:  pkgconfig(modulemd) >= %{libmodulemd_version}
 BuildRequires:  pkgconfig(smartcols)
 BuildRequires:  gettext
+BuildRequires:  gpgme-devel
```

Comment:
not `pkgconfig(gpgme)`?

Reply:
There is no gpgme.pc file in gpgme-devel package.